### DISCUSSION TITLE
Run Milvus without Zilliz account or any internet connect?

### DISCUSSION THREAD

**Discussion by CHesketh76** [milvus-io/milvus#31667] [Q&A and General discussion]
Can I run this without signing up for an account? I want to run this on a local machine which strictly has no internet connection.

**Comment by yiwangdr** [comment]
Yes, you can install milvus. There are different ways to install it e.g. https://milvus.io/docs/install_standalone-docker.md

**Reply by CHesketh76** [reply]
Is there a non-docker option?

**Reply by yiwangdr** [reply]
Yes, there are tabs of different install options from the link above , e.g. https://milvus.io/docs/install_standalone-aptyum.md
